File: week-2/exercicios/fatura.cpp
#include<iostream>

int main() {
  char nome[10], mes[10];
  int dia;
  float valor;

  std::cout << "Digite o nome do cliente: ";
  std::cin >> nome;

  std::cout << "Digite o dia de vencimento: ";
  std::cin >> dia;

  std::cout << "Digit o mes de vencimento: ";
  std::cin >> mes;

  std::cout << "Digite o valor da fatura: ";
  std::cin >> valor;

  std::cout << "Ola, " << nome <<"\n";
  std::cout << "A sua fatura com vencimento em " << dia;
  std::cout << " de " << mes;
  std::cout << " no valor de R$ " << valor;
  std::cout << " esta fechada." << std::endl; 
  
  return 0;
}

File: week-2/exercicios/fatura.py
"""
Exercirio 3
Uma empresa de cartão de crédito envia suas faturas por email com a seguinte mensagem.

Olá, Fulano de Tal
A sua fatura com vencimento em 9 de Janeiro no valor de R$ 350,00 esta fechada.
"""
nome = input("Digite o nome do cliente:")
dia = input("Digite o dia de vencimento:")
mes = input("Digite o mês de vencimento:")
valor = input("Digite o valor da fatura:")

print('Olá,', nome)
print('A sua fatura com vencimento em', dia, 'de', mes, 'no valor de R$',valor,'está fechada.')
